src/ethaniccc/Mockingbird/cheat/other/EditionFaker.php

```diff
@@ -31,6 +31,10 @@ class EditionFaker extends Cheat implements Blatant{
     private $fakers = [];
 
     public function __construct(Mockingbird $plugin, string $cheatName, string $cheatType, bool $enabled = true){
+	    if(!$this->getServer()->getOnlineMode()){
+		    $this->getPlugin()->getLogger()->emergency('EditionFaker MUST need online mode');
```

Comment:
Change this to debug, since Poggit might reject this for unnecessary messages in startup, instead put this warning in the config as a note.